template/web/wp-content/themes/studiometa/app/Managers/ACFManager.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Boostrap ACF
+ *
+ * @package Studiometa
+ */
+
+namespace Studiometa\Managers;
+
+/** Class **/
+class ACFManager {
```

Comment:
What about having a `ManagerInterface` defining the `run` method?

Ping @ptt-homme @perruche @lusimeon

Reply:
I guess it's probably a good idea, just a basic implementation like this is working, is it enough ? My experience with interfaces is pretty much non-existant

```php
/**
 * Interface for Managers
 */
namespace Studiometa\Interfaces;

/** Interface **/
interface ManagerInterface {
	public function run();
}
```

Reply:
I think the namespace should be `Studiometa\Managers` as it is the interface for the manager classes. And we can add a documentation block before the `run` method, it will be readable by IDE and tools such as PHPStan.

**app/Managers/ManagerInterface.php**
```php
namespace Studiometa\Managers;

interface ManagerInterface {
  /**
   * ...
   * @return {void}
   */
  public function run();
}
```

**app/Managers/ThemeManager.php**
```php
namespace Studiometa\Managers;

class ThemeManager implements ManagerInterface {
  /**
   * @inheritdoc
   */
  public function run() {}
}
```